src/librustc_resolve/lifetimes.rs

```diff
@@ -155,12 +155,14 @@ struct NamedRegionMap {
 
 crate enum MissingLifetimeSpot<'tcx> {
     Generics(&'tcx hir::Generics<'tcx>),
-    HRLT { span: Span, span_type: HRLTSpanType },
+    HRLT { span: Span, span_type: ForLifetimeSpanType },
```

Comment:
`HRLT` needs to be updated now with the changed terminology.
I would suggest calling the variant `HigherRanked`.